### DISCUSSION TITLE
How to fetch the events that is related to specific categories with GQL .

### DISCUSSION THREAD

**Discussion by Brijeshgrg1999** [solspace/craft-calendar#312] [Feature Requests]
**Describe the bug or issue you're experiencing**
I am trying to fetch the event's data using graph ql since i am using calendar plugin but I want to fetch the events that is related to some of the categories for the events 

**Steps to reproduce**

1. These are my categories setuped 
 
<img width="1086" alt="Screenshot 2024-05-20 at 12 38 38 PM" src="https://github.com/solspace/craft-calendar/assets/118506279/f036267e-ba80-45aa-bf39-800c0536b78d">

2. This is how i have added categories in the event entry ( in the Event Category) 

<img width="1375" alt="Screenshot 2024-05-20 at 12 40 18 PM" src="https://github.com/solspace/craft-calendar/assets/118506279/87332f16-5556-404d-9774-24f89961562c">

3 . This is how i am trying to fetch using this query 
<img width="916" alt="Screenshot 2024-05-20 at 12 40 55 PM" src="https://github.com/solspace/craft-calendar/assets/118506279/2d8ca242-306a-4647-a7a8-8bc15cf9eeb0">

can someone help me how can i  fetch the events that are related to only some of the categories .
let's say if I want to fetch the events that is related to " Cocktails & WineTasting " , how would I do it ? 
Thank you 

**Craft & Plugin Info (please complete the following information):**

- Craft Version: craft pro 4.8.9
- Calendar Version: 5.0.3

**Comment by kjmartens** [comment]
This currently isn't possible as we don't support filtering events by categories in GQL in Calendar. We've noted this for future consideration, however. 🙂
